specs/scoring_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'simplecov'
+SimpleCov.start
+
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+require_relative '../lib/scoring'
+Minitest::Reporters.use!
+
+describe "Scoring class" do
+  it "Creates new instance of Scoring class" do
+    Scrabble::Scoring.new.class.must_equal Scrabble::Scoring
```

Comment:
I'm not sure if this test is necessary - since you're not supposed to create an instance of `Scrabble::Scoring`, it doesn't make sense to test that behavior.